pkg/losetup/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM linuxkit/alpine:3683c9a66cd4da40bd7d6c7da599b2dcd738b559 AS mirror
+RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
+RUN apk add --no-cache --initdb -p /out  \
+    alpine-baselayout \
+    busybox
+
+# Remove apk residuals
+RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
+
+FROM scratch
+ENTRYPOINT []
+WORKDIR /
+COPY --from=mirror /out/ /
+
+COPY loopy.sh /loopy.sh
```

Comment:
Same comment as with `crypto.sh`. I think this should just be 

```Dockerfile
COPY loopy.sh loopy
```

This requires changing it everywhere else in this PR, but search-and-replace should make it easy.

Reply:
BTW, I love that you called it "loopy". :-)

Reply:
was inspired by "mountie" :)
fixed as well